homeassistant/components/fritzbox/config_flow.py

```diff
@@ -31,7 +31,7 @@
     }
 )
 
-RESULT_AUTH_FAILED = "auth_failed"
+RESULT_INVALID_AUTH = "invalid_auth"
 RESULT_NOT_FOUND = "not_found"
```

Comment:
Please change this constant as well
```suggestion
RESULT_NO_DEVICES_FOUND = "not_found"
```